### DISCUSSION TITLE
pages/about

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#6] [Announcements]
# pages/about

https://blog.yasking.org/pages/about.html

**Comment by twoconk** [comment]
交换个友链 谢谢 https://it3q.com

**Reply by sincerefly** [reply]
已添加。

**Reply by twoconk** [reply]
感谢!

**Comment by lzh1230** [comment]
站名：博客驿站
链接：https://www.bloginn.vip/
图标：https://www.bloginn.vip/img/bklnntx.png
介绍：发现新博客认识新朋友！

**Reply by sincerefly** [reply]
你好 👋 欢迎来访，已添加友链

**Comment by moeleak** [comment]
前几天刚给朋友转发说想体验地球online就去玩一下外卖游戏，没想到真有人去尝试了hhh
话说能交换友联吗
https://leak.moe

**Reply by sincerefly** [reply]
hhha，已添加友链，你的博客很酷呀！

**Reply by moeleak** [reply]
你的也是！感觉很干净。

**Comment by sysalong** [comment]
博主估计也得30岁了吧，咋又想去背景了啊，很累。。。。真的很累。

**Reply by sincerefly** [reply]
主要是回到老家发现职业成长空间比较受限，那会儿还单身，权衡来看，就趁着还能出来的时候出来工作。
（PS：2017 年的北京雾霾情况，我感觉再不跑人都可能没了）

**Reply by mvpbang** [reply]
习惯大城市节奏，回老家适应不了的。等干不动在回去，有钞票哪里都可以去，深圳没雾霾

**Reply by sincerefly** [reply]
是的，很难适应，深圳挺好，就是离我的老家太远了，横跨南北 🤦‍♂️

**Comment by mvpbang** [comment]
交换个友链 谢谢: https://blog.mvpbang.com/

**Reply by sincerefly** [reply]
添加好了！🤝

**Comment by zhyong26** [comment]
交换下友链，谢谢，已添加贵站 https://zhyong.site/links/

站点名称：材料与逻辑
站点网址：https://zhyong.site/
站点描述：从原子到结构，从数据到洞察
LOGO：https://i.zhyong.site/i/2026/02/17/6994661354dc2.png

**Reply by sincerefly** [reply]
哈喽，👋 已添加～

**Reply by zhyong26** [reply]
好的，会经常来逛逛
